creator-node/scripts/run-tests.sh

```diff
@@ -3,15 +3,56 @@
 set -o xtrace
 set -e
 
+IPFS_CONTAINER=cn-test-ipfs-node
+DB_CONTAINER='cn_test_db'
+REDIS_CONTAINER='cn_test_redis'
+
 PG_PORT=$POSTGRES_TEST_PORT
 if [ -z "${PG_PORT}" ]; then
   PG_PORT=4432
```

Comment:
i think this may be conflicting with my local cn1 postgres
```
+ docker run -d --name cn_test_db -p 127.0.0.1:4432:5432 postgres:11.1
7409ed833a329f770649d16c58ddd724c373f34d21dc40cad2951599c4d75238
docker: Error response from daemon: driver failed programming external connectivity on endpoint cn_test_db (3421a846332491f249ef9eada3c3cba6ec3c25580c92ee0c42e2a603b92aa57f): Error starting userland proxy: listen tcp 127.0.0.1:4432: bind: address already in use.
npm ERR! Test failed.  See above for more details.
```